.gitignore

```diff
@@ -0,0 +1,11 @@
+build/*
+!build/
+.vscode/*
```

Comment:
I guess so. otherwise it could stay, so who ever uses vscode doesn't need to remove it manually and it could grow by further gitignors related to other IDEs.

Reply:
Done

Reply:
At the moment, the vscode-related part is still there. can you please remove it? :)